server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_21__add_kill_non_support_version.sql

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+INSERT INTO `config_system_configuration` (`key`, `value`, `application`, `profile`, `label`, `description`)
+VALUES ('non_support_kill_min_version', '4.2.5', 'odc', 'default', 'master', 'min version of non support kill session') ON DUPLICATE KEY UPDATE `id`=`id`;
```

Comment:
1. Follow config_system_configuration's key style, key name may be like `odc.session.kill-query-or-session.max-ob-version`?
2. Is it more appropriate to put the SQL in `data.sql`, in case your code relies on the refreshable config?

Reply:
1、That's more reasonable ~
2、The absence of this configuration means that the OB version is not restricted, and placing this configuration in data.sql will always insert this configuration.